build-configs/integreat-test-cms/index.ts

```diff
@@ -5,14 +5,9 @@ import {
   WebBuildConfigType
 } from '../BuildConfigType'
 import integreatPlatformBuildConfigs from '../integreat'
+import { template } from '../integreat/assets/cityNotCooperatingTemplate'
```

Comment:
Not sure if we shouldn't use a symlink here as well. Or you can just do the following below:
```
    featureFlags: {
      ...,
     cityNotCooperatingTemplate: integreatPlatformBuildConfigs.common.featureFlags.cityNotCooperatingTemplate
    }
```

Reply:
good enough as it is